src/js/AuthAPI.js

```diff
@@ -0,0 +1,33 @@
+import User from './User';
+import axiosInstance from './config';
+
+export default class AuthAPI {
+  static login({ identifier, password }) {
+    console.log(axiosInstance.headers);
```

Comment:
Please remove this unnecessary console log

Reply:
done